UserDefinedReductions/user_defined_reductions.cpp

```diff
@@ -0,0 +1,68 @@
+// RUN: %clangxx -fsycl %s -o %t.out
```

Comment:
Should be under SYCL/ director, I believe.

Reply:
Thanks for finding, accidentally created a folder in wrong directory.